contents/docs/data-warehouse/sql/variables.mdx

```diff
@@ -8,44 +8,63 @@ availability:
   enterprise: full
 ---
 
-export const WarehouseLight = "https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_02_24_at_14_05_39_2x_5a280bacfd.png"
-export const WarehouseDark = "https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_02_24_at_14_05_53_2x_32a95cbe79.png"
+SQL variables enable you to dynamically set values in your queries. 
 
-SQL variables enable you to create configurable, reusable queries by defining placeholders that can be dynamically replaced with different values.
+## Creating SQL variables
 
-## Using variables in SQL queries
+To create a variable, go to the [SQL editor](https://app.posthog.com/sql), click the **Variables** tab, click **Add variable**, and choose your variable type (string, number, date, list, or boolean). Enter the variable name and value(s) and click **Save** and you'll be able to use it in your queries.
```

Comment:
How are variables scoped? It looks like they're not per query, are they like, project wide?

Reply:
project, yes